src/Framework/App.Runtime/src/Microsoft.AspNetCore.App.Runtime.csproj

```diff
@@ -461,6 +464,43 @@ This package is an internal implementation of the .NET Core SDK and is not meant
           StandardOutputImportance="High" />
   </Target>
 
+  <Target Name="_GenerateComposites"
+    DependsOnTargets="_ExpandRuntimePackageRoot">
+
+    <!-- These were used for debugging and log analyzing. Will be removed when
+         the PR is merged.
+
+    <Message Text="MESSAGEY: IN GENERATE COMPOSITES TARGET RULE" Importance="high" />
+    <Message Text="MESSAGEY RUNTIME PACKAGE ROOT: $(RuntimePackageRoot)" Importance="high" />
+    <Message Text="MESSAGEY TARGET DIR: $(TargetDir)" Importance="high" />
+    <Message Text="MESSAGEY TARGET ARCHITECTURE: $(TargetArchitecture)" Importance="high" />
+    -->
+
+    <PropertyGroup>
+      <CompositeTargetDir>$(TargetDir)Composite\</CompositeTargetDir>
+      <RemoveDir Directories="$(CompositeTargetDir)" />
+      <MakeDir Directories="$(CompositeTargetDir)" />
```

Comment:
This & the `RemoveDir` above are tasks and can't be invoked from inside of the `PropertyGroup` tag they're currently in, they should be moved to below the closing of the `PropertyGroup` and above the `Exec` below